buildenv/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# (C) Copyright IBM Corporation 2016, 2021
+# (C) Copyright IBM Corporation 2016, 2022
```

Comment:
Please update the copyright notice with the text used everywhere else.

Reply:
Done, good spot

Reply:
I see the dates have changed, but I still see non-standard text.

Reply:
Yes, I only looked at the years. Taking a closer look, the rest of the text is quite different from the text in other files, which is odd. I see from the history that it was the same originally but changed when a new Dockerfile was created to eventually replace the original. The new legal text was probably an inadvertent change (it's not mentioned in the update). On that basis, I've changed it back to match the other files.

Reply:
This file must continue to use the comment style understood by docker (lines beginning with `#`).

Reply:
Argh. Glad you spotted that because I wouldn't have (clearly)! Now fixed.